xblockmufi/public/edit.css

```diff
@@ -0,0 +1,196 @@
+@font-face {
+  font-family: "Andron Scriptor Web";
+  src: url('fonts/AndronScriptorWeb.eot');
+  src: url('fonts/AndronScriptorWeb.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */ url('fonts/AndronScriptorWeb.woff') format('woff'), /* Pretty Modern Browsers */ url('fonts/AndronScriptorWeb.ttf') format('truetype'), /* Safari, Android, iOS */;
```

Comment:
trailing comma!

Reply:
Fixed: needed to run `make static --always-make`